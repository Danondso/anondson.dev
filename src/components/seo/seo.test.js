import React from 'react';
import { useStaticQuery } from 'gatsby';
import renderer from 'react-test-renderer';
import SEO from './seo';

describe('SEO Test Suite', () => {
  it('renders correctly', () => {
    useStaticQuery.mockReturnValueOnce({
      site: {
        siteMetadata: {
          description: 'A Site Description For A Test',
        },
      },
    });
    const tree = renderer.create(<SEO title="Test" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
