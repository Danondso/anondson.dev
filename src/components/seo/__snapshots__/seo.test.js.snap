// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEO Test Suite renders correctly 1`] = `null`;
